zerver/tests/fixtures/config.generate_data.json

```diff
@@ -35,6 +35,8 @@
         "\n```python\ndef concat(*args, sep='/'):\n    return sep.join(args)\n```",
         "\n```\n(map + [1 2 3] [4 5 6])\n ;;=> (5 7 9)\n```"
       ],
+      "embed-links":
+      ["https://www.youtube.com", "https://www.twitch.com", "https://www.twitter.com", "https://www.netflix.com", "https://www.amazon.com"],
```

Comment:
`embed-links` is not an ideal name -- I think `urls-in-topics`  would be much more clear.

My other thought is that it'd probably be better to use more realistic URLs for individual pages:
* An article on the Zulip blog
* a GitHub pull request.
* A GitHub issue
* A direct link to a tweet from the Zulip twitter account
* a page from Zulip's ReadTheDocs

(People aren't likely to use the URL of a major brand's homepage in their topic).